tensorflow_addons/layers/noisy_dense.py

```diff
@@ -208,23 +219,27 @@ def bias(self):
         if self.use_bias:
             return self.mu_bias + (self.sigma_bias * self.eps_bias)
 
+    def _scaled_noise(self, size, dtype):
+        x = tf.random.normal(shape=size, dtype=dtype)
+        return tf.sign(x) * tf.sqrt(tf.abs(x))
+
```

Comment:
Doesn't need to be a class function since dtype is being passed.

Reply:
Okej, I change it.